### ISSUE TITLE
Tests for a fresh new addon fail

### ISSUE THREAD

**Issue by bugant** [ember-cli/ember-cli#5350]
Tests for a newly created addon fail.

Repro steps:

- ember addon ember-cli-foo
- cd ember-cli-foo
- ember test --server

These are the reported errors:

```
Uncaught Error: Assertion Failed: The tests file was not loaded. Make sure your tests index.html includes "assets/tests.js". at http://localhost:7357/assets/vendor.js, line 16745
Uncaught Error: Could not find module `ember/resolver` imported from `dummy/tests/helpers/resolver` at http://localhost:7357/assets/vendor.js, line 146
```

**Comment by bugant** [comment]
This is a duplicate of #5316.

**Comment by eddiesholl** [comment]
Hey @bugant I'm hitting the same issue on ember 1.13.13, any ideas on a workaround or fix? Don't quite follow how #5316 is related here, isn't that a jQuery dependency issue?

**Comment by bugant** [comment]
@eddiesholl it is indeed a jQuery dependency issue. Indeed if you pin jQuery to 1.11 in your `bower.json` everything will be OK.
